sky/shell/platform/ios/framework/Source/FlutterViewController.mm

```diff
@@ -153,20 +163,21 @@ - (void)connectToEngineAndLoad {
                                ? VMTypePrecompilation
                                : VMTypeInterpreter;
 
-  [_dartProject launchInEngine:_engine
-                embedderVMType:type
-                        result:^(BOOL success, NSString* message) {
-                          if (!success) {
-                            UIAlertView* alert = [[UIAlertView alloc]
-                                    initWithTitle:@"Launch Error"
-                                          message:message
-                                         delegate:nil
-                                cancelButtonTitle:@"OK"
-                                otherButtonTitles:nil];
-                            [alert show];
-                            [alert release];
-                          }
-                        }];
+  [_dartProject
+      launchInEngine:_engine
+      embedderVMType:type
+              result:^(BOOL success, NSString* message) {
+              if (!success) {
+                UIAlertView* alert = [[UIAlertView alloc]
+                        initWithTitle:@"Launch Error"
+                              message:message
+                             delegate:[[LaunchErrorDelegate alloc] init]
```

Comment:
You are leaking the delegate. Besides, you don't need one anyway. You can create a class extension to `FlutterViewController` that implements the `UIAlertViewDelegate`, add the method in its body and specify `self` as the delegate.